gradle/libs.versions.toml

```
@@ -5,7 +5,9 @@ gradle-dependency-handler-extensions = "1.1.0"
 kotlin-core = "1.8.21"
 kotlin-ktlint-gradle = "11.4.2"
 kotlin-ktlint-source = "0.49.1"
+kotlin-plugin = "1.7.20"
```

어떤 플러그인의 버전인지 명시하는게 좋을 거 같아!

```suggestion
kotlin-plugin-serialization = "1.7.20"
```

---

좋은 지적입니다~